runtime/oti/phelpers.m4

```diff
@@ -135,7 +135,10 @@ define({FUNC_PTR},{r11})
 
 define({CALL_INDIRECT},{
 	ifelse($1,,,{mr FUNC_PTR,$1})
-	bl ._ptrgl[pr]
+	laddr r2,0(FUNC_PTR)
+	mtctr r2
+	laddr r2,eval(ALen)(FUNC_PTR)
```

Comment:
what is `eval(ALen)`?

Reply:
It's an m4ism - ALen is a macro defined to a number, so if Alen is 4, Alen(reg) evaluates to 4. I'll change Alen to the fancy constant macro that handles argument properly.